sample-app/src/main/java/com/joinforage/android/example/ui/pos/ui/ComposableForagePANEditText.kt

```diff
@@ -0,0 +1,38 @@
+package com.joinforage.android.example.ui.pos.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.viewinterop.AndroidView
+import com.joinforage.android.example.ui.pos.network.AUTH_TOKEN
+import com.joinforage.forage.android.ui.ForageConfig
+import com.joinforage.forage.android.ui.ForagePANEditText
+
+@Composable
+fun ComposableForagePANEditText(
+    merchantId: String,
+    withPanElementReference: (element: ForagePANEditText) -> Unit
+) {
+    AndroidView(
+        factory = { context ->
+            ForagePANEditText(context).apply {
+                this.setForageConfig(
+                    ForageConfig(
+                        merchantId = merchantId,
+                        sessionToken = AUTH_TOKEN
+                    )
+                )
+                this.requestFocus()
+                withPanElementReference(this)
```

Comment:
Yep, that's exactly it! Since we have to pass in the reference to the actual view when tokenizing the card, this was the best way I could think of do it. Compose doesn't really have the concept of grabbing a specific view by an ID, since manually manipulating views isn't really a thing you should be doing in this paradigm.